### PR TITLE
[Config] don't break on virtual stack frames in ClassExistenceResource

### LINKED ISSUE #34665: Script cache: clear returned with error code 1 (In FileLoader.php and ClassExistenceResource.php)
**Symfony version(s) affected**: x.y.z

**Description** 

I am running the demo creation command "symfony new my_project_name --demo" and this is the result. Screenshot: http://g2f.nl/05q0363

 "symfony new my_project_name --full" works however. 

**How to reproduce**  

I can't follow the mentioned guidelines: It happens when you run "symfony new my_project_name --demo"

**Possible Solution**  

I don't know, I started discovering this framework today.

**Additional context**  

Screenshot: http://g2f.nl/05q0363

The only thing i have changed is enabling the pdo_mysql and pdo_sqlite extension in php.ini. file. This is a pure standard installation. Oh I remember now I have changed the realpath_cache_size to 5242880 as recommended by the installer.

I hope this helps...

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #34665
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by stof** [comment]
this is the same issue than https://github.com/symfony/symfony/pull/34636 but with a fix targetting the right lowest branch, right ? If yes, why is the fix different ?

**Comment by nicolas-grekas** [comment]
That's a different class, not the same job/code.

File: src/Symfony/Component/Config/Resource/ClassExistenceResource.php [hunk 1]
```diff
@@ -191,15 +191,17 @@ public static function throwOnRequiredClass($class, \Exception $previous = null)
             }
 
             $props = [
-                'file' => $trace[$i]['file'],
-                'line' => $trace[$i]['line'],
+                'file' => isset($trace[$i]['file']) ? $trace[$i]['file'] : null,
```
Comment (ahmedash95) [inline src/Symfony/Component/Config/Resource/ClassExistenceResource.php]: I'm just wondering why not using `$trace[$i]['file'] ?? null` instead? is it for backword comptability with PHP 5.6?

Comment (ahmedash95) [reply src/Symfony/Component/Config/Resource/ClassExistenceResource.php]: I see it's 3.4 with support for php 5.4 👍

**Pull request merged**